opsgenie/resource_opsgenie_team_test.go

```diff
@@ -155,10 +156,10 @@ resource "opsgenie_team" "test" {
 `, rString)
 }
 
-func testAccOpsGenieTeam_complete(rString string) string {
+func testAccOpsGenieTeam_complete(randomUser, randomTeam string) string {
 	return fmt.Sprintf(`
 resource "opsgenie_user" "test" {
-  username  = "genietest@opsgenie.com"
+  username  = "genietest-%s@opsgenie.com"
```

Comment:
👋 heads up after writing the original provider with this approach, the API introduced a limit on the number of users who can be invited per-account - as such it may be necessary to get the test account configured to avoid this limit so that these tests can run nightly?

Reply:
Hey yes i saw the logs and currently waiting the upgrade of that account. This pr just to make sure for future or any parallel acctest.